packages/core/src/constants.js

```diff
@@ -23,7 +23,12 @@ const RENDER_ONLY_METHODS = [
   'authentication.oauth1Config.authorizeUrl',
 ];
 
-const REQUEST_OBJECT_SHORTHAND_OPTIONS = { isShorthand: true, replace: true };
+const REPLACE_CURLIES = Symbol('replaceCurlies');
+
+const REQUEST_OBJECT_SHORTHAND_OPTIONS = {
+  isShorthand: true,
```

Comment:
This isn't referenced anywhere, so we can nuke this.

But perhaps we better use something like `IS_INTERNAL_SHORTHAND = Symbol('isInternalShorthand')` to also for ourselves be more clear?

Reply:
Deleted in [9735102](https://github.com/zapier/zapier-platform/pull/1032/commits/973510225b1f5dbcba908438101e1aae974047cc).